### DISCUSSION TITLE
Question with e3nn of ARES

### DISCUSSION THREAD

**Discussion by Wes-V3** [e3nn/e3nn#340] [Q&A] [answered]
I have followed the readme to install e3nn, and it turns out I have successfully installed it.
But when I run ARES program, I got the error:
`ModuleNotFoundError: No module named 'e3nn.point'`

I also tried to follow ARES readme to install e3nn.

> Install e3nn
> 
> Install the ARES-compatible version of e3nn (note this version is only provided
> for compatability, further development should be done using the main e3nn branch):
> 
> `pip install git+https://github.com/drorlab/e3nn_ares.git@6682f60c2c4d99375cfa6321dfcdfa87f104cf8b#egg=e3nn`

but I only got the error:

```
(ares) xx:~$ pip install git+https://github.com/drorlab/e3nn_ares.git@6682f60c2c4d99375cfa6321dfcdfa87f104cf8b#egg=e3nn
Collecting e3nn
  Cloning https://github.com/drorlab/e3nn_ares.git (to revision 6682f60c2c4d99375cfa6321dfcdfa87f104cf8b) to /tmp/pip-install-h08ah9de/e3nn_bac3d3844eca4ceba88ec89c0cd39a84
  Running command git clone --filter=blob:none --quiet https://github.com/drorlab/e3nn_ares.git /tmp/pip-install-h08ah9de/e3nn_bac3d3844eca4ceba88ec89c0cd39a84
Username for 'https://github.com': Wes-V3
Password for 'https://Wes-V3@github.com': 
  remote: Repository not found.
  fatal: repository 'https://github.com/drorlab/e3nn_ares.git/' not found
  error: subprocess-exited-with-error
  
  × git clone --filter=blob:none --quiet https://github.com/drorlab/e3nn_ares.git /tmp/pip-install-h08ah9de/e3nn_bac3d3844eca4ceba88ec89c0cd39a84 did not run successfully.
  │ exit code: 128
  ╰─> See above for output.
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
error: subprocess-exited-with-error

× git clone --filter=blob:none --quiet https://github.com/drorlab/e3nn_ares.git /tmp/pip-install-h08ah9de/e3nn_bac3d3844eca4ceba88ec89c0cd39a84 did not run successfully.
│ exit code: 128
╰─> See above for output.

note: This error originates from a subprocess, and is likely not a problem with pip.
```

Since they didn't release their code on github, but zonodo instead. So I come here to ask for help. Thank you!

**Comment by Wes-V3** [comment, marked as answer]
I'm sorry that I found the solution. It's because they have a modified version on zonodo.
I want to delete this question but it doesn't seem possible.
Sorry for the bothering!

**Comment by yuehua-Song666** [comment]
Hi,

Sorry for disturbing you.
I got the same problem, but I don't really understand 'they have a modified version on zonodo' this part. Can you please tell me how you solved this?

Thank you,
Yuehua
